client/connection.go

```diff
@@ -259,6 +262,11 @@ func ConnectSimpleStreams(url string, args *ConnectionArgs) (ImageServer, error)
 		args = &ConnectionArgs{}
 	}
 
+	// Do not include modern post-quantum curves in the ClientHello to avoid
+	// compatibility issues (connection resets) with broken middleboxes that
+	// can't handle large ClientHello messages split over multiple TCP packets.
+	args.LegacyCurvesOnly =	true
```

Comment:
Shall we move this to be setting to true in the call site of `ConnectSimpleStreams` rather than overriding something could have been potentially set in `args`?

Reply:
There are 3 call sites which would cause some comment duplication.

I was debating if this flag should be part of the `args` struct to begin with. If an environment requires this quirk, it needs it all the time so there should be no need to vary it per call site.

My initial version simply passed the right value to the `tlsHTTPClient()` instead. Maybe that would be best?

Reply:
> My initial version simply passed the right value to the tlsHTTPClient() instead. Maybe that would be best?

Sounds good